apps/token-e2e/src/integration/flow/governance-flow.cy.js

```diff
@@ -230,11 +230,7 @@ context('Governance flow - with eth and vega wallets connected', function () {
         let proposalId = proposal.response.body.data.busEvents[0].event.id;
         cy.get('[data-testid="set-proposals-filter-visible"]').click();
         cy.get('[data-testid="filter-input"]').type(proposerId);
-        cy.get(`#${proposalId}`).should(
-          'contain',
-          `Freeform proposal: ${proposalId}`,
-          txTimeout
-        );
+        cy.get(`#${proposalId}`).should('contain', `${proposalId}`, txTimeout);
```

Comment:
I believe `txTimeout` in the assertion should go in the `cy.get` as that's where it gets retried

Reply:
Well spotted - i have removed them entirely since not required